Esignet/scripts/Esignet_Helper_Script.jmx

```diff
@@ -161,7 +165,11 @@
           <hashTree/>
         </hashTree>
 
+        <HTTPSamplerProxy guiclass="HttpTestSampleGui" testclass="HTTPSamplerProxy" testname="Mobile - Authentication API" enabled="true">
+
+
         <HTTPSamplerProxy guiclass="HttpTestSampleGui" testclass="HTTPSamplerProxy" testname="Authentication API" enabled="true">
+
```

Comment:
this file is not loading, there is some issue because of merge, please check

Reply:
I have resolved the conflicts. Could you pls check